packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export type CardHeaderProps = {
+  headingWithDefaultServiceFtlId: string;
+  headingWithCustomServiceFtlId: string;
+  headingText: string;
+  serviceName?: string;
+};
+
+const CardHeader = ({
+  headingWithDefaultServiceFtlId,
+  headingWithCustomServiceFtlId,
+  headingText,
+  serviceName,
+}: CardHeaderProps) => {
+  return (
+    <header>
+      {serviceName ? (
+        <FtlMsg id={headingWithCustomServiceFtlId} vars={{ serviceName }}>
+          <h1 className="card-header">
+            {headingText}{' '}
+            <span className="card-subheader">{`to continue to ${serviceName}`}</span>
+          </h1>
+        </FtlMsg>
+      ) : (
+        <FtlMsg id={headingWithDefaultServiceFtlId}>
+          <h1 className="card-header">
+            {headingText}{' '}
+            <span className="card-subheader">
+              {`to continue to account settings`}
```

Comment:
> I should have asked this before but should this be capitalized since it is now? I'm guessing not with the whole accounts vs Accounts decision, just doubly checking.

I think it might have been an error in the first place to have this capitalized since it's not a product/brand/service. 'account' singular was lowercase in the settings brand terms, even before the recent decision. Unless this is used in a title (title case), it should probably be lowercase - please correct me if I'm wrong!
 
> Btw you don't have to have "{`" here, see in `LinkAbout` for example where we just place text inside a span without backticks.

👀

Reply:
> I think it might have been an error in the first place to have this capitalized since it's not a product/brand/service. 'account' singular was lowercase in the settings brand terms, even before the recent decision. Unless this is used in a title (title case), it should probably be lowercase - please correct me if I'm wrong!

I'm unsure honestly but it seems right? We could double check with Vesta at some point about it.